elixir/config/config.exs

```diff
@@ -243,6 +243,11 @@ config :tailwind,
     cd: Path.expand("../apps/web/assets", __DIR__)
   ]
 
+config :workos, WorkOS.Client,
```

Comment:
nit: May want to use some kind of static string here like the other configs, even if it fails. Or use get_env!

Configuration tends to grow unwieldy when reading lots of env vars. I think that was one of the reason @AndrewDryga started the config system we have now.

Reply:
Yeah, I've missed the configs part in the review. We need something that is `good enough for dev env` in `config.exs`, right now that would be just `nil`. Maybe we can either omit it here and move `api_key` and `client_id` to `dev.exs`?

Reply:
I'll need to check to make sure everything works in dev mode if you don't supply a valid set of WorkOS creds.  I think it should be fine so long as you don't need to add a JumpCloud provider and if you do need to for some reason, you'll need to get a WorkOS staging API key to use, but I will test it out.